primitives/application-crypto/src/traits.rs

```diff
@@ -31,7 +31,7 @@ use sp_std::{fmt::Debug, vec::Vec};
 /// Typically, the implementers of this trait are its associated types themselves.
 /// This provides a convenient way to access generic information about the scheme
 /// given any of the associated types.
-pub trait AppCrypto: 'static + Send + Sync + Sized + CryptoType + Clone {
```

Comment:
Why?

Reply:
`AppCrypto` is required to be implemented by any `Pair` and thus for the new `bandersnatch::Pair` as well.

Unfortunately `bandersnatch_vrf::SecretKey` embeds the `SecretScalar` ([here](https://github.com/w3f/ring-vrf/blob/4957177a717c7555c8df2869012201017b62e66b/dleq_vrf/src/keys.rs#L86)) AND `SecretScalar` is defined to contain an `UnsafeCell` ([here](https://github.com/w3f/ring-vrf/blob/4957177a717c7555c8df2869012201017b62e66b/ark-secret-scalar/src/lib.rs#L78)). Thus can't be `Sync`.

Nevertheless, `Sync` is not a hard requirement for our internals. So far `Send` seems sufficient for all our use cases